packages/venia-concept/src/components/PurchaseDetailsPage/PurchaseDetails/purchaseDetails.js

```diff
@@ -50,6 +69,7 @@ class PurchaseDetails extends Component {
                     classes={{ root: classes.orderDetailsBlockRoot }}
                 />
                 <OrderItemsList
+                    withBorders
```

Comment:
There is no need to pass this to list component. In general, I have added a comment describing how we can remove this property at all.

Reply:
Again, remove this.